library/src/mixins/property/property_file_split/usign_property_regex_fy.py

```diff
@@ -39,4 +36,12 @@ def _property_file_split(self) -> List[str]:
             len(property_file_split) == 6
         ), f"Property file split length {len(property_file_split)} is invalid"
 
-        return property_file_split
+        property_file_split_model = PropertyFileSplitModel(
+            user_imports=property_file_split[0],
+            property_name=property_file_split[2],
+            implementation_name=property_file_split[4],
+            property_type=property_file_split[3],
+            mixin_split=property_file_split[5].split("\n"),
```

Comment:
Let's keep the original string so if anyone needs it for a different purposes, like parsing, they can use it. This way no one has the original string.

Move split to the place where it is consumed.